.editorconfig

```diff
@@ -0,0 +1,22 @@
+root = true
+
+[*]
+indent_style = tab
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.{json,js,jsx,html,css,sh,styl,ts,tsx,yml,yaml}]
+indent_style = space
+indent_size = 2
+
+[*.{ts,tsx}]
+max_line_length = 100
+
+[.eslintrc]
+indent_style = space
+indent_size = 2
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
Our team usually gitignores editor files, I know some don't- just wanted to make sure you meant to check this in. ✅

Reply:
Good catch, thanks!